src/main/java/com/example/demo/service/FileUploadService.java

```diff
@@ -0,0 +1,55 @@
+package com.example.demo.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Service
+public class FileUploadService {
+    private String path = "//home//ivan//Завантаження//Нова тека//Нова тека//";
```

Comment:
Use Lombok's `FieldDefaults` annotation for setting properties' access modifiers